DEN/donations.yml

```diff
@@ -0,0 +1,66 @@
+---
+
+- ticker: DEN
+  recipient: 
+    name: Perth Childrens Hospital Foundation
+    address: PO Box 8249, Subiaco East, WA, 6008, Australia
+    tel: "+62 (08) 6456 2222"
+    contact: info@pchf.org.au
+  ada: 34
+  usd: 41.82
+  timestamp: 1619395680
+  files:
+    - DEN/pchf1.pdf
```

Comment:
The files are not showing the recipient of the transaction. So it's hard to validate if the money really went to this organization purely based on the PDF. Do you have any external reachable link which could be used for validation? Want to avoid that you need to create a new set of PDFs ;)

Reply:
apologies, i forgot to change the timestamps for newest donations. I will resubmit!